integration/PhpSpec/Console/Prompter/DialogTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace integration\PhpSpec\Console\Prompter;
+
+use PhpSpec\Console\Prompter\Dialog;
+
+/**
+ * @requires function \Symfony\Component\Console\Helper\DialogHelper::askConfirmation
+ */
+class DialogTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var \Symfony\Component\Console\Output\OutputInterface
+     */
+    private $output;
+
+    /**
+     * @var \Symfony\Component\Console\Helper\DialogHelper
+     */
+    private $dialogHelper;
+
+    /**
+     * @var \PhpSpec\Console\Prompter
+     */
+    private $prompter;
+
+    function setUp()
+    {
+        $this->output = $this->getMock('Symfony\Component\Console\Output\OutputInterface');
+        $this->dialogHelper = $this->getMockBuilder('Symfony\Component\Console\Helper\DialogHelper')
+                                   ->disableOriginalConstructor()->getMock();
```

Comment:
I didn't want to use the extension when 4.5 is just around the corner. I'll convert to prophecy when 4.5 is stable.